src/main/kotlin/org/rust/ide/injected/RsDoctestLanguageInjector.kt

```diff
@@ -145,7 +145,7 @@ class RsDoctestLanguageInjector : MultiHostInjector {
                 }
                 if (index == cratesEndIndex && !alreadyHasMain) {
                     if (prefix == null) prefix = StringBuilder()
-                    prefix.append("fn main() {")
+                    prefix.append("fn IntellijIdeaRulezzz() {")
```

Comment:
Is it intentional change?

Reply:
Yes, it disables folding in this case:
![image](https://user-images.githubusercontent.com/6079006/60921498-5327d880-a2a3-11e9-92c6-fc0b33245ef8.png)

Reply:
Consider using another name to not confuse it with the one from completion, and also extract it to some constant.

Reply:
@vlad20012 Fixed.